build.gradle.kts

```diff
@@ -39,10 +40,10 @@ dependencies {
 
     intellijPlatform {
         phpstorm(properties("platformVersion"))
-        pluginVerifier("1.307")
+        pluginVerifier("1.373")
         zipSigner()
         instrumentationTools()
-        testFramework(TestFrameworkType.Platform.JUnit4)
+        testFramework(TestFrameworkType.Platform)
```

Comment:
Could you please elaborate on this change?
You added `testImplementation("junit:junit:4.13.2")` so we are still using junit4?

Reply:
The rename is per https://github.com/JetBrains/intellij-platform-gradle-plugin/releases/tag/v2.0.0-beta7

The junit was added as per the breaking changes note on https://github.com/JetBrains/intellij-platform-gradle-plugin/releases/tag/v2.0.0-RC1